requests_mock/mocker.pyi

```diff
@@ -34,100 +40,172 @@ class MockerCore:
       method: Union[str, AnyMatcher],
       url: Union[str, Pattern[str], AnyMatcher],
       response_list: Optional[List[Dict[str, Any]]] = ...,
+      *,
       request_headers: Dict[str, str] = ...,
       complete_qs: bool = ...,
       status_code: int = ...,
-      text: str = ...,
-      headers: Optional[Dict[str, str]] = ...,
+      reason: str = ...,
+      headers: Dict[str, str] = ...,
+      cookies: Union[CookieJar, Dict[str, str]] = ...,
+      json: Union[Any, Callable[[_RequestObjectProxy, _Context], Any]] = ...,
+      text: Union[str, Callable[[_RequestObjectProxy, _Context], str]] = ...,
+      content: Union[bytes, Callable[[_RequestObjectProxy, _Context], bytes]] = ...,
+      body: Union[IOBase, Callable[[_RequestObjectProxy, _Context], IOBase]] = ...,
+      raw: HTTPResponse = ...,
```

Comment:
interesting - i always thought this was callable as well. Guess noone has ever used that.